homeassistant/components/switchbot_cloud/climate.py

```diff
@@ -83,22 +90,51 @@ class SwitchBotCloudAirConditioner(SwitchBotCloudEntity, ClimateEntity):
     _attr_precision = 1
     _attr_name = None
 
+    async def async_added_to_hass(self) -> None:
+        """Run when entity about to be added."""
+        await super().async_added_to_hass()
+
+        if not (last_state := await self.async_get_last_state()):
```

Comment:
I think we need to check for unavailable and unknown too:
```suggestion
        if (
            not (last_state := await self.async_get_last_state())
            or state.state in (STATE_UNAVAILABLE, STATE_UNKNOWN)
        ):
```